frontend/src/components/db/QueryPageTopStack/ButtonsRenderer.tsx

```diff
@@ -0,0 +1,65 @@
+import * as React from "react";
+import SettingsIcon from "@mui/icons-material/Settings";
+import { Box, useMediaQuery } from "@mui/material";
+import IconButton from "@mui/material/IconButton";
+import Popover from "@mui/material/Popover";
+import { useTheme } from "@mui/material/styles";
+
+export const buttonWrapperStyles = {
+  display: "flex",
+  flexDirection: { xs: "column", md: "row" },
+  alignItems: { md: "center" },
+  gap: 1,
+};
+
+export const ButtonsRenderer = ({
```

Comment:
what's a `Renderer`? Can we just call this `Buttons`? I would also suggest a more specific name like `QueryPageButtons` or something